<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login</class>
 <widget class="QMainWindow" name="login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>567</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1088</width>
    <height>567</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1088</width>
    <height>567</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>login</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(25, 31, 50);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="cyber_bg">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>411</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wardenfort 1.png</pixmap>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>20</y>
      <width>333</width>
      <height>527</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>333</width>
      <height>527</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>333</width>
      <height>527</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(44, 60, 75, 0.6); 
border-radius: 50px;
</string>
    </property>
    <widget class="QLabel" name="username_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #ffffff;
background-color: rgba(0, 0, 0, 0);
}


</string>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="typeUN_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>275</width>
       <height>52</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>275</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>275</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
padding: 50px;
	color: rgb(255, 255, 255);
}

</string>
     </property>
     <property name="placeholderText">
      <string>Type your username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="typePASS_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>275</width>
       <height>52</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>275</width>
       <height>52</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>275</width>
       <height>52</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
border-radius: 15px; 
background-color: rgb(44, 60, 75);
padding: 50px;
	color: rgb(255, 255, 255);
}
</string>
     </property>
     <property name="placeholderText">
      <string>Type your password</string>
     </property>
    </widget>
    <widget class="QLabel" name="pass_text">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #ffffff;
background-color: rgba(0, 0, 0, 0);
}


</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>400</y>
       <width>275</width>
       <height>34</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>275</width>
       <height>34</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>275</width>
       <height>34</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
border-radius: 15px; 
background-color: rgb(22, 27, 46);
color: white;
}
QPushButton:hover {
}
QPushButton:pressed {
background-color: rgb(24, 40, 55); /* Adjust pressed color if needed */
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QLabel" name="forgotpass_text">
     <property name="geometry">
      <rect>
       <x>217</x>
       <y>340</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>8</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
	font: 7.5pt &quot;Segoe UI&quot;;
    color: #ffffff;
background-color: rgba(0, 0, 0, 0);
}


</string>
     </property>
     <property name="text">
      <string>Forgot Password?</string>
     </property>
    </widget>
    <zorder>typeUN_box</zorder>
    <zorder>username_text</zorder>
    <zorder>typePASS_box</zorder>
    <zorder>pass_text</zorder>
    <zorder>loginButton</zorder>
    <zorder>forgotpass_text</zorder>
   </widget>
   <widget class="QLabel" name="wf_logo">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>40</y>
      <width>101</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 0, 0, 0);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo (5).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="wf_word_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>140</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgba(0, 0, 0, 0);
background-color: rgba(0, 0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>C:/Users/admin/Downloads/logo (4).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="user_icon">
    <property name="geometry">
     <rect>
      <x>406</x>
      <y>231</y>
      <width>30</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pass_icon (11).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="pass_icon">
    <property name="geometry">
     <rect>
      <x>408</x>
      <y>310</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>pass_icon (4).png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="signup_button">
    <property name="geometry">
     <rect>
      <x>511</x>
      <y>459</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>8</pointsize>
      <italic>false</italic>
      <bold>false</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 7.5pt &quot;Segoe UI&quot;;
text-decoration: underline;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QLabel" name="wf_word">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wardenfort_word (1).png</pixmap>
    </property>
   </widget>
   <widget class="QCheckBox" name="eye_open">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>327</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
    border: none; /* Remove border */
}

QCheckBox {
    background-color: transparent; /* Remove background color */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>eye_open.png</normaloff>eye_open.png</iconset>
    </property>
    <property name="shortcut">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="eye_closed">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>326</y>
      <width>40</width>
      <height>22</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox::indicator {
    border: none; /* Remove border */
}

QCheckBox {
    background-color: transparent; /* Remove background color */
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>eye_closed.png</normaloff>eye_closed.png</iconset>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
